No such comm: c4a8ccca54ab4fe5a058d12f15e6caf0
No such comm: c78694a1f81646298067b2f494e64a14
No such comm: d704b6ab9c8142b794f9d65af673f3f1
No such comm: 7e1073520db047dc90c29b7e3970e59a
No such comm: 3296aed545df4f4ca34c0fc1178e56d5
No such comm: a5a507547f80450db8fd0c46f6378b12
No such comm: 7d0b36e50b4846f48153812486fd1dff
No such comm: 39056e41caa04fea93d3b3d73ee1a176
No such comm: 8d4caf9b4e2e44eeaeefdd870e4b9da4
No such comm: 2392fc4a0a134789b2a109746f121fee
No such comm: cb8e94d78f744236b04792d8acec3185
No such comm: 5f6a9163fd2e494e96ac0ac905051ce8
No such comm: 34b316989fa9443da2b829d91980fcf0
No such comm: fd0e5d909b384b369de601fae113c4c7
No such comm: 12a125aab44a4693bc11995bd638b98b
No such comm: 1c14519a0f9a455fa2483656bb395368
Exception in thread SystemMonitor:
Traceback (most recent call last):
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/system_monitor.py", line 118, in _start
    asset.start()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/cpu.py", line 166, in start
    self.metrics_monitor.start()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/interfaces.py", line 168, in start
    logger.info(f"Started {self._process.name}")
AttributeError: 'NoneType' object has no attribute 'name'
Exception in thread SystemMonitor:
Traceback (most recent call last):
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/system_monitor.py", line 118, in _start
    asset.start()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/cpu.py", line 166, in start
    self.metrics_monitor.start()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/interfaces.py", line 168, in start
    logger.info(f"Started {self._process.name}")
AttributeError: 'NoneType' object has no attribute 'name'
Exception in thread SystemMonitor:
Traceback (most recent call last):
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/system_monitor.py", line 118, in _start
    asset.start()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/cpu.py", line 166, in start
    self.metrics_monitor.start()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/interfaces.py", line 168, in start
    logger.info(f"Started {self._process.name}")
AttributeError: 'NoneType' object has no attribute 'name'
Exception in thread SystemMonitor:
Traceback (most recent call last):
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/system_monitor.py", line 118, in _start
    asset.start()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/cpu.py", line 166, in start
    self.metrics_monitor.start()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/interfaces.py", line 168, in start
    logger.info(f"Started {self._process.name}")
AttributeError: 'NoneType' object has no attribute 'name'
Exception in thread SystemMonitor:
Traceback (most recent call last):
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/system_monitor.py", line 118, in _start
    asset.start()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/cpu.py", line 166, in start
    self.metrics_monitor.start()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/interfaces.py", line 168, in start
    logger.info(f"Started {self._process.name}")
AttributeError: 'NoneType' object has no attribute 'name'
No such comm: efb8ea7f290549b08b6664da4b2ec4cb
No such comm: 514b1dd57d6343d5ad2b51fb8a367d75
No such comm: e9a831343cee493884fde40c8e75da6d
No such comm: db6d346bbbef4039a62f6a14a9c90094
No such comm: 4c28d884e8f1421eb6bce2fa68ab773f
No such comm: fd583366ffbb457b809a526d7877a107
No such comm: 9bdfc102dc7640408284bca76f64bec0
No such comm: dad0263acad34ba8978a73232d4edb36
No such comm: ea6da49441124926b544468b4fb72c6d
No such comm: fbed586fc1974a8595bef402bc7a55f7
No such comm: 52ef5d25576c4d97bf88007bfe96f371
2023-01-13 21:11:47.887708: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-01-13 21:11:47.896284: I tensorflow/core/common_runtime/process_util.cc:146] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
Exception in thread SystemMonitor:
Traceback (most recent call last):
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/system_monitor.py", line 118, in _start
    asset.start()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/cpu.py", line 166, in start
    self.metrics_monitor.start()
  File "/home/dahala/miniconda3/envs/mlashok/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/interfaces.py", line 168, in start
    logger.info(f"Started {self._process.name}")
AttributeError: 'NoneType' object has no attribute 'name'
Epoch 1/100

















